src/components/Auth/SignUpFirstForm/index.tsx

```diff
@@ -1,46 +1,59 @@
-import React, {useEffect} from 'react';
-import {useTranslation} from "react-i18next";
-import {useForm} from "react-hook-form";
-import {yupResolver} from "@hookform/resolvers/yup";
-import { useGoogleLogin } from '@react-oauth/google';
-import {IconButton, InputAdornment } from '@mui/material';
+import React, { useEffect, useState } from 'react';
```

Comment:
SignUpFirstForm why there is first and not first form. rename or make it more generic

Reply:
In our project we have two registration steps. Therefore, for this we created two components - "SignUpFirstStepForm" and "SignUpSecondStepForm". I also put these two components in a shared folder "SignUpForm" for convenience:
![image](https://user-images.githubusercontent.com/91757681/232059776-35c682ff-47ae-4d88-be24-edc82c3cd419.png)